### DISCUSSION TITLE
Batch of requests

### DISCUSSION THREAD

**Discussion by Moltik** [Netflix/dgs-framework#71] [Q&A]
Hello. Does the framework provide the ability to handle batch requests? For example.
```
[
  {
    "operationName":"shows",
    "variables":{
    "titleFilter":"Stranger Things"
     },
    "query":"query shows($titleFilter: String) { shows(titleFilter:$titleFilter){title releaseYear } }"
  },
  {
    "operationName":"shows",
    "variables":{
    "titleFilter":"Ozark"
    },
  "query":"query shows($titleFilter: String) { shows(titleFilter:$titleFilter){title releaseYear } }"
  }
]
```

**Comment by paulbakker** [comment]
No there's no support for handling multiple queries. What would be the use case though?

**Reply by Moltik** [reply]
We use the apollo client on our pages, which gives us the ability to send an array of requests. https://www.apollographql.com/blog/batching-client-graphql-queries-a685f5bcd41b/ and with the kickstart library, we can handle such requests. 
Since we are considering replacing it with your framework, it is important for us to be able to handle such requests as well.
